docs/docs/development/tests.md

```diff
@@ -4,7 +4,7 @@ sidebar_label: Tests
 ---
 
 - Running tests requires [native posix support](posix-board.md).
-- Any folder under `/app/tests` containing `native_posix.keymap` will be selected when running `west test`.
+- Any folder under `/app/tests` containing `native_posix_64.keymap` will be selected when running `west test`.
```

Comment:
Unrelated to this PR, but `west test` has never worked on Windows because it uses a `.sh` for running the test from the `test.py` file.